transformers4rec/torch/features/tabular.py

```diff
@@ -175,7 +175,7 @@ def from_schema(  # type: ignore
         output = cls(
             continuous_module=maybe_continuous_module,
             categorical_module=maybe_categorical_module,
-            text_embedding_module=None,
+            text_embedding_module=kwargs.pop("text_embedding_module", None),
```

Comment:
how is `text_embedding_module` propogated later on in embedding layer? do we concat them? how do we create the lookups?

Reply:
It will be an additional module to the `Embeddings` and `Continuous` layer. So the module will expect as inputs the dictionary of raw features and should include all the processing logic needed to represent text features in its forward method. (such as filtering text features, performing embedding lookup (or not), and normalization). The output of this module will be aggregated with the output of `Embeddings` and `Continuous` layers based on the `aggregation` specified in `tr.TabularSequenceFeatures.from_schema(...)`

Reply:
As the `text_embedding_module` is not specific to text, what about renaming it to something like `custom_module`?